gitコマンドの基本操作についてまとめたテキストを追加

yusuke-0528 opened the pull request:
お疲れ様です。
gitの各種コマンドについてまとめたテキストを作成しました。
お手すきの際に確認のほどよろしくお願いいたします。

yusk2004 committed:
gitコマンドの基本操作についてまとめたテキストを追加

お疲れ様です。
gitの各種コマンドについてまとめたテキストを作成しました。
お手すきの際に確認のほどよろしくお願いいたします。

ka-mui reviewed (commented):
@yusuke-0528 
他のPRのコメントを参考に修正してください。

yusk2004 committed:
gitコマンドの基本操作についてまとめたテキストを修正

お疲れ様です。
gitの各種コマンドについてまとめたテキストの空行についての修正を行いました。
修正の確認よろしくお願いします。

yusk2004 committed:
gitコマンドの基本操作についてまとめたテキストの空行を修正

お疲れ様です。
gitの各種コマンドをまとめたテキストの空行について不要であると考えたため修正を行いました。
お手すきの際に確認のほどよろしくお願いいたします。

ka-mui reviewed (commented):
@yusuke-0528 
ファイル名のprefixについているgit-というのは必要でしょうか。
